libraries/stdlib/samples/test/samples/number/int.kt

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2010-2017 JetBrains s.r.o.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package samples.number
+
+import samples.*
+
+class Int {
```

Comment:
I don't expect there will be distinct samples for each primitive type. Also naming this class the same as the built-in `Int` looks dangerous.

So if you're going to improve further the samples for `coerce*` functions, this can be named as `Coercion`. 
Also it's worth to find a more general package name, since not only numbers could be coerced in range, but also any Comparable type could.

Reply:
> I don't expect there will be distinct samples for each primitive type

OK, then for the class name, how about `ComparableOps`? In that case we could add samples that are not `coercion` (e.g. `minOf`, `maxOf`)

I'm not sure for the package name. Could you add the sample code under `sample` package? I thought I need to have at least one package.

Reply:
`samples.comparisons.ComparableOps` is fine too.